doc/Extensions/RRDCached.md

```diff
@@ -291,5 +291,54 @@ Depending on many factors, you should see the Ops/sec drop by ~30-40%.
 
 ## Securing RRCached
 
-Please see [RRDCached Security](RRDCached-Security.md)
+According to the [man page](https://linux.die.net/man/1/rrdcached),
+under "SECURITY CONSIDERATIONS", rrdcached has no authentication or
```

Comment:
I still don't get this, can't we just use a firewall?

Reply:
I just copied the old page

Reply:
Is this even needed on a single node setup? When using distributed polling there is also the SQL-connection which should probably be secured. I think there should be a notice on distributed setup guide that if the connectivity between servers is not trusted, setup a PtP-tunnel with something like WireGuard and use that tunnel for the server traffic. Normal polling should not use the tunnel.

Reply:
Maybe the security-section should have it's own page (own section instead of being under the misc) and all specific information about securing the installation should go under there in one place? In my opinion the basic installation guide should just provide a sane basic configuration without exposing anything other than the web-server to the outside.

Reply:
I'd be in favor with summary deletion of this page and creation of a new page with security recomendations.

Reply:
Feel free to push

Reply:
I don't plan on changing more, if  someone wants push, feel free.